packages/db/src/migrations/20240609000711_phrase_gloss_events/migration.sql

```diff
@@ -0,0 +1,43 @@
+-- DropForeignKey
+ALTER TABLE "GlossHistoryEntry" DROP CONSTRAINT "GlossHistoryEntry_languageId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "GlossHistoryEntry" DROP CONSTRAINT "GlossHistoryEntry_userId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "GlossHistoryEntry" DROP CONSTRAINT "GlossHistoryEntry_wordId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "GlossHistoryEntry" DROP CONSTRAINT "GlossHistoryEntry_pkey";
+
+-- Rename table
+ALTER TABLE "GlossHistoryEntry" RENAME TO "GlossEvent";
+
+ALTER TABLE "GlossEvent"
+  ADD COLUMN "id" SERIAL PRIMARY KEY,
+  ADD COLUMN "phraseId" INTEGER;
+
+UPDATE "GlossEvent"
+  SET "phraseId" = data."phraseId"
+FROM (
+  SELECT e."languageId", e."wordId", ph.id AS "phraseId"
```

Comment:
This should set the phrase IDs of all gloss events before making it a required field